MODS.js

```diff
@@ -15,7 +15,7 @@
 	"inRepository": true,
 	"translatorType": 3,
 	"browserSupport": "gcsv",
```

Comment:
@adam3smith While we are looking a this translator, can/should we activate all browserSupport?

Reply:
sure -- does browserSupport actually do anything on import translators, though?

Reply:
Hm... the testing page looks different if `v` is missing, but otherwise maybe there is no consequence. However, I would still prefer to have them more consistently everywhere.